scripts/setup.sh

```
@@ -91,30 +111,30 @@ fi
+rm -rf .env
 python -m venv .env
 source .env/bin/activate
 pip install --upgrade pip
 pip install -r requirements.txt
 deactivate
-cd ..
 
 # ---- Set up the sync-microservice ----
 echo -e "${YELLOW}Setting up sync-microservice...${NC}"
-cd sync-microservice
+cd "$REPO_ROOT/sync-microservice"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)